src/App.css

```diff
@@ -1,3 +1,15 @@
+@import "color.css";
+
+* {
```

Comment:
You might one to add here also  the pseudo-elements before and after to have the same box-sizing everywhere 
https://css-tricks.com/box-sizing/#universal-box-sizing